modules/asc_swift/1.0.0rc1/presubmit.yml

```diff
@@ -0,0 +1,11 @@
+matrix:
+  platform: ['macos', 'ubuntu2004']
+  bazel: ['7.x']
+
+tasks:
+  run_tests:
+    name: Test workspace
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    test_targets:
+      - '//...'
```

Comment:
@aaronsky, this should probably be `@asc_swift//...`

Reply:
thanks, will take a look. sorry for the repeated failing PRs, they can all be closed. i swear i tested this before i cut a tag this time – is there a way for me to approximate the presubmit process in my own CI?

Reply:
The current setup is tied to Bazel CI, so it's not trivial, but I'll investigate more into this later since more users are encountering this.

For now, how we test a module is documented at https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#presubmit

Reply:
Right now I have a presubmit.yml that's pretty close to rules_swift's, and yet it appears as though my `environment` configuration is not interpolating with the other environment variables in the same way as rules_swift's pipeline is. I'm guessing it's a silly YAML thing, but I'm not sure what else could be the issue.

Reply:
I've figured it out, and I believe it's a bug in the presubmit script. When using a `bcr_test_module`, task `environment` fields don't get expanded. If you don't do that, then it works as expected.